renovation_core_graphql/reports/sdls/root.graphql

```diff
@@ -0,0 +1,25 @@
+scalar RenovationReportFilterInput
+scalar ReportRowResult
+
+type GetReportResult{
+    result : [ReportRowResult]
+    columns: [ReportColumn]
+    message: String,
+    chart: String,
+    report_summary: String,
+    skip_total_row: Int,
+    status: String,
+    execution_time: Float,
+    add_total_row: Int,
+}
+
+scalar ExportReportCustomColumn
+
+enum ExportFileFormatType{
+    EXCEL
+}
+
+extend type Query {
+    getRenovationReport(report: String!, filter: RenovationReportFilterInput): GetReportResult
```

Comment:
@e-lobo How is `RenovationReportFilterInput` supposed to be sent ?

Reply:
@assemmarwan this will work `{from_date: "2021-05-01", to_date: "2021-05-31", group_by: YEAR}`